src/pages/questionPage.js

```diff
@@ -24,6 +26,7 @@ export const initQuestionPage = () => {
     answersListElement.appendChild(answerElement);
     answerElement.addEventListener('click', showAnswer);
 
+    // eslint-disable-next-line no-inner-declarations
```

Comment:
Do you need this eslint rule?

Reply:
I do not but without it, all the function is red underlined and it is added automatically.